doc/preview.yml

```
@@ -0,0 +1,57 @@
+site:
+  title: Neo4j Graph Data Science
+  url: https://neo4j.com/docs
+  start_page: graph-data-science:ROOT:index.adoc
+
+content:
+  sources:
+  - url: ../../
+    start_path: public/doc
+    branches: 
+    - HEAD
+    include: public/doc/
```

```suggestion
  - url: ../
    start_path: doc
    branches: 
    - HEAD
    include: doc/
```

These changes should fix the error you are having when running `npm run start`

---

works like a charm, thanks!